### PR TITLE
Add Anscombe Quartet Example

### PR DESCRIPTION
**Pull request by mcorrell**
Anscombe Quartet example

### CODE REVIEW

**Comment by kanitw** [comment]
@mcorrell  Can you rename this to `trellis_anscombe` and add it to https://github.com/vega/vega-lite/blob/master/examples/vl-examples.json#L166 so that it shows up in the editor?

File: examples/specs/anscombe.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
+  "description": "Anscombe's Quarter (https://en.wikipedia.org/wiki/Anscombe%27s_quartet)",
+  "data": {
+    "values": [
+      {"Series":"I", "X":10.0, "Y":8.04},
```
Comment (domoritz) [inline examples/specs/anscombe.json]: I think it's worth putting this dataset in https://github.com/vega/vega-datasets

**Comment by mcorrell** [comment]
Okay, will add to [vega-datasets](https://github.com/vega/vega-datasets ), and then change this example to use the url source + update name once that's done

**Comment by mcorrell** [comment]
Okay, added a json of anscombe data to vega/vega-datasets#6

**Comment by domoritz** [comment]
I just released a new version of vega-datasets with the data.

File: examples/vl-examples.json [hunk 2]
```diff
@@ -164,6 +164,12 @@
     }
   ],
   "Trellis": [
+    { "name": "trellis_anscombe",
+      "title": "Anscombe's Quarter",
+      "galleryParameters": {
+        "backgroundSize": 130%"
```
Comment (domoritz) [inline examples/vl-examples.json]: invalid json

**Comment by mcorrell** [comment]
Should be "Quartet" rather than "Quarter." Also seems to not compile correctly in the gallery.

**Comment by domoritz** [comment]
Yeah, we have to push a new version of the editor.

**Pull request merged**